Header header

int64 edge_front_map_mnid
int64 edge_back_map_mnid

cloud_edge_slam/KeyFrame[] key_frames
cloud_edge_slam/MapPoint[] map_points
